src/api/goods.ts

```diff
@@ -6,6 +6,12 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => (
+  getAll()
+    .then(goods => goods.slice(0, 5))
```

Comment:
```suggestion
    .then(goods => goods.slice(0, 5))
```
Посортуй, бо завдання не повністю виконане

Reply:
уже)